BREAKING.md

```diff
@@ -44,6 +45,9 @@ The `loader` property from `MockFluidDataStoreContext` class was deprecated in r
 ### `maxMessageSize` removed from `IConnectionDetails` and `IDocumentDeltaConnection`
 The `maxMessageSize` property from `IConnectionDetails` and `IDocumentDeltaConnection` was deprecated in 0.51, and has now been removed from the `container-definitions` and `driver-definitions` packages respectively. To replace its functionality, use `serviceConfiguration.maxMessageSize`.
 
+### Remove `IntervalCollection.getView()` from sequence dds
+The `IntervalCollection.getView()` was deprecated and no longer reference anywhere.
```

Comment:
For breaking change notices, the goal is to advise folks whether they are affected and what they need to do in response to the change if so.  So e.g.

```suggestion
The `IntervalCollection.getView()` was removed.  If you were calling this API, you should instead refer to the `IntervalCollection` itself directly in places where you were using the view.
```